sci-libs/Rtree/Rtree-0.8.3_p20180802.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+EGIT_COMMIT_HASH="c6437a8ae182cb58aef1d0a4465bfc0c6f75b273"
+
+DESCRIPTION="R-Tree spatial index for Python GIS"
+HOMEPAGE="https://github.com/Toblerity/rtree"
+SRC_URI="https://github.com/Toblerity/rtree/archive/${EGIT_COMMIT_HASH}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="sci-libs/libspatialindex
+	sci-libs/scipy"
```

Comment:
fix your tabs

Reply:
what's wrong with them?

Reply:
@Zlogene I double-checked in my editor, there's only two tabs and they both use the correct character.

Reply:
@TheChymera github/git uses ts=8 (if we speak vim language), here I see your `sci-libs/scipy` is shifted

Reply:
ah, I see, sounds like an aesthetic problem that cannot be fixed for all viewers, which is why I have one indent level, since that is as close as you can get to semantically settling this.

Reply:
I see it's indented using single tab, so that is correct.